Traceback (most recent call last):
  File "snippet.py", line 11, in <module>
    model_filename = sys.argv[1]
IndexError: list index out of range
